test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -261,7 +271,7 @@ private void UpdateCliWithLatestNuGetAssemblies(string cliDirectory)
             foreach (var nupkgName in nupkgsToCopy) {
                 using (var nupkg = new PackageArchiveReader(FindMostRecentNupkg(nupkgsDirectory, nupkgName)))
                 {
-                    var files = nupkg.GetFiles()
+                     var files = nupkg.GetFiles()
                     .Where(fileName => fileName.StartsWith("lib/netstandard")
                                     || fileName.StartsWith("lib/netcoreapp")
                                     || fileName.Contains("NuGet.targets"));
```

Comment:
If there're two folders(eg: netstandard2.0 & netstandard2.1), we only need to copy the dll under netstandard2.1, right?